javaagent-tooling/build.gradle.kts

```diff
@@ -37,6 +37,18 @@ dependencies {
 
   implementation("io.opentelemetry.contrib:opentelemetry-aws-xray-propagator")
 
+  implementation("io.opentelemetry.contrib:opentelemetry-aws-resources") {
+    exclude("io.opentelemetry", "opentelemetry-api")
+    exclude("io.opentelemetry", "opentelemetry-sdk")
+    exclude("io.opentelemetry.semconv", "opentelemetry-semconv")
```

Comment:
why is this necessary?

Reply:
it was necessary when I added these to a distro - but I'm not sure what the exact problem was

Reply:
I double checked - it doesn't make a difference in our case.